File: src/main/java/client/database/data/MakerCreateData.java
package client.database.data;

public class MakerCreateData {
   private int requiredLevel;

   private int requiredMakerLevel;

   private int requiredMeso;

   private int quantity;

   public MakerCreateData(int requiredLevel, int requiredMakerLevel, int requiredMeso, int quantity) {
      this.requiredLevel = requiredLevel;
      this.requiredMakerLevel = requiredMakerLevel;
      this.requiredMeso = requiredMeso;
      this.quantity = quantity;
   }

   public int getRequiredLevel() {
      return requiredLevel;
   }

   public int getRequiredMakerLevel() {
      return requiredMakerLevel;
   }

   public int getRequiredMeso() {
      return requiredMeso;
   }

   public int getQuantity() {
      return quantity;
   }
}

File: engine/model/src/main/scala/client/database/data/MakerCreateData.scala
package client.database.data

class MakerCreateData(private var _requiredLevel: Int, private var _requiredMakerLevel: Int,
                      private var _requiredMeso: Int, private var _quantity: Int) {
  def requiredLevel: Int = _requiredLevel

  def requiredMakerLevel: Int = _requiredMakerLevel

  def requiredMeso: Int = _requiredMeso

  def quantity: Int = _quantity
}
